WordPressKitTests/WordPressComRestApiTests+AsyncAwait.swift

```diff
@@ -0,0 +1,94 @@
+import XCTest
+import OHHTTPStubs
+import WordPressShared
+
+@testable import WordPressKit
+
+@available(iOS 15.0.0, *)
+class WordPressComRestApiAsyncAwaitTests: XCTestCase {
+
+    private let url = "https://public-api.wordpress.com/rest/v1.1/sites/0/media/"
+
+    override func setUp() {
+        super.setUp()
+    }
+
+    override func tearDown() {
+        super.tearDown()
```

Comment:
Nitpick: In WordPress iOS, we [decided to remove the calls to `super`](https://github.com/wordpress-mobile/WordPress-iOS/issues/17569) for setup and tear down as they don't do anything, apparently.

```suggestion
    override func tearDown() {
```

Reply:
TIL! Fixed. :)